File: Data-Structures-and-Algorithms-using-Swift/AlgorithmsChapters/CA/IsPrime.swift
import Foundation

func isPrime(_ n : Int) -> Bool {
	var answer : Bool = (n > 1) ? true : false
	var i : Int = 2
	while (i * i <= n) {
		if (n % i == 0) {
			answer = false
			break
		}
		i += 1
	}
	return answer
}

// Testing code.
print(isPrime(8))
print(isPrime(11))
/* 
false
true
 */

File: Data-Structures-Algorithms-In-TS/AlgorithmsChapters/CA/IsPrime.ts
function isPrime(n: number): boolean {
	let answer: boolean = n > 1;
	for (let i = 2; i * i <= n; ++i) {
		if (n % i === 0) {
			answer = false;
			break;
		}
	}
	return answer;
}

console.log(isPrime(8));
console.log(isPrime(11));


/*
false
true
*/